zebra-state/src/service/finalized_state/zebra_db/shielded.rs

```diff
@@ -178,6 +178,44 @@ impl ZebraDb {
         Some(Arc::new(tree))
     }
 
+    /// Returns the Sapling note commitment tree of the finalized tip
+    /// or the empty tree if the state is empty.
```

Comment:
This should be refactored.

Reply:
Fixed in https://github.com/ZcashFoundation/zebra/pull/7334/commits/1bd86bcb3cdbcf2326318d5157c2ea10dd1b453e.